src/main/java/com/api/service/NetworkingService.java

```diff
@@ -30,17 +36,22 @@ public NetworkingService(WebClient webClient) {
     }
 
     public List<SxRocket> getRockets(Integer limit, Integer offset) {
+
+        System.out.println("Start getRockets()");
```

Comment:
usually use log4j to log the operation, where you can dictate the level of logging